### PR TITLE
Devices API - remove request params from new operation

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Removes request params from the new "List all Users for a Device" operation
- **Is this PR related to a Monolith release?** no

### Resolves:

* [OKTA-610267](https://oktainc.atlassian.net/browse/OKTA-610267)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/devices/index.md](https://github.com/okta/okta-developer-docs/blob/94011e86b750e777bf6242555a8c3fe8842f5d8f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fdevices%2Findex.md) | [71](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9193352f-d871-4972-88b9-2ecd0e8b56b2) | 48  | 96  | 82  | 56  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.1cc7929f-aa54-4640-b5ac-9cf6adc1c7b5?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by jane-kuang-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/devices/index.md [hunk 1]
```diff
@@ -612,17 +612,11 @@ curl -v -X GET \
 
 Lists all [Users](/docs/reference/api/users/#user-object) for a Device by `deviceId`
 
-#### Request parameters
+#### Request parameter
 
 | Parameter | Description                                                | ParamType | DataType | Required | Default |
 | --------- | ---------------------------------------------------------- | --------- | -------- | -------- | ------- |
-| after     | Specifies the pagination cursor for the next page of users | Query     | String   | FALSE    |         |
 | id        | ID of the Device                                          | URL       | String   | TRUE     |         |
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/devices/index.md:619]: Do you want to change this parameter name to match the method description and the request example?
```suggestion
| deviceId        | ID of the Device                                          | URL       | String   | TRUE     |         |
```

**Review by vanngo-okta — APPROVED** [review]
LGTM. Just a nit suggestion.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/devices/index.md](https://github.com/okta/okta-developer-docs/blob/bc7a53e7de516aa2f1f1a3d0c4c54da6ab34cfeb/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fdevices%2Findex.md) | [72](https://okta.acrolinx.cloud/api/v1/checking/scorecards/edd0081e-3df1-4660-963a-19810f840718) | 48  | 100  | 82  | 56  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.500646dc-0904-457f-9cdf-4c6b59fd7aaf?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Pull request merged**